jum.py

```diff
@@ -35,48 +35,26 @@ def __init__(self):
         self.smiley = utils.load_image('smiley.png', True)
 
     def setup(self):
-        self.obj_grid = []
-        for i in range(10000):

```

Comment:
This loop makes self.total items in the list self.obj_grid, with each item ranging from 1 to self.n